platform/qemu.go

```diff
@@ -183,13 +201,12 @@ func (qc *QEMUCluster) GetDiscoveryURL(size int) (string, error) {
 
 // Copy the base image to a new nameless temporary file.
 // cp is used since it supports sparse and reflink.
-func setupDisk(imageFile string) (*os.File, error) {
+func setupDisk(imageFile string) (string, error) {
 	dstFile, err := ioutil.TempFile("", "mantle-qemu")
 	if err != nil {
-		return nil, err
+		return "Error: fail to create the destination file", err
```

Comment:
Do not stick error messages into file path, either `return "", err` or `return "", fmt.Errorf("something exitra: %v", err)`